data/mods/gen3/abilities.ts

```diff
@@ -55,6 +55,15 @@ export const Abilities: {[k: string]: ModdedAbilityData} = {
 		inherit: true,
 		flags: {},
 	},
+	hustle: {
+		inherit: true,
+		onSourceModifyAccuracy(accuracy, target, source, move) {
+			const physicalTypes = ['Normal', 'Fighting', 'Flying', 'Poison', 'Ground', 'Rock', 'Bug', 'Ghost', 'Steel'];
+			if (move.category === 'Physical' || physicalTypes.includes(move.type)) {
```

Comment:
You can drop the category check, you'll produce the same list either way. One that you need to add back is ``typeof accuracy === 'number'`` so you can't miss things like Block or Helping Hand.